src/main/java/com/github/wovnio/wovnjava/HtmlChecker.java

```diff
@@ -24,6 +24,16 @@ public boolean canTranslateContent(String html) {
         return isHtml(head) && !isAmp(head);
     }
 
+    public boolean isTextContentType(String type) {
+        String contentType = type.toLowerCase();
+
```

Comment:
@bstewart00  I think of it this way, but what do you think?

```suggestion
    public boolean canUtf8ConvertableContentType(String type) {
        String contentType = type.toLowerCase();

        if (contentType.contains("text/css")) {
            return false;
        }

```

Reply:
This is too difficult to handle now, don't worry about it 😄 
we _do_ need to use utf-8 for css if the css was indeed utf-8. But if they used SHIFT-JIS for example, we would need to use that. However, wovnjava assumes all text responses are UTF-8, so we can't really do this now.

Reply:
Well, thank you!
I'll leave you with just a comment as a future concern.